rollup.config.js

```diff
@@ -21,6 +36,7 @@ const getBabelOptions = (targets) => {
   }
   return {
     ...config,
+    comments: false,
     extensions,
   }
 }
```

Comment:
As we only use babel for cjs, let's make it simple and consistent with jotai/valtio.
```js
const getBabelOptions = (targets) => {
  const config = createBabelConfig({ env: (env) => env === 'build' }, targets)
  return {
    ...config,
    plugins: [
      ...config.plugins,
      ['@babel/plugin-transform-runtime', { helpers: true, regenerator: true }],
    ],
    babelHelpers: 'runtime',
    extensions,
    comments: false,
  }
}
```

I tried removing `'@babel/plugin-transform-regenerator'` and the result is the same. What do you think?

Reply:
I noticed you put @babel/plugin-transform-runtime in babel.config.js in jotai. Maybe we can do the same in zustand.